prow/cmd/artifactuploader/BUILD.bazel

```diff
@@ -26,7 +26,7 @@ prow_image(
 )
 
 go_binary(
-    name = "artifact-uploader",
+    name = "artifactuploader",
```

Comment:
Can we move the artifactuploader rename into its own PR (and asssign stevekuznetsov)?